README.md

```diff
@@ -77,8 +77,6 @@ Otherwise, to get an APK, do:
 
 ### Build release version of app
 
-First, run `yarn update-version` to copy the `package.json` version into the native project files, and to assign a unique build number.
-
```

Comment:
We should delete the section header, now that it's empty.

Reply:
I think it's still relevant as it differentiates building a release version vs a debug version like the header above. Otherwise we'd have iOS, Android, iOS, Android all under the 

### Run the app in debug mode

header.